crates/rattler_conda_types/src/channel/mod.rs

```diff
@@ -85,16 +88,17 @@ impl NamedChannelOrUrl {
     }
 
     /// Converts the channel to a base url using the given configuration.
+    /// This method ensures that the base url always ends with a `/`.
     pub fn into_base_url(self, config: &ChannelConfig) -> Url {
         match self {
             NamedChannelOrUrl::Name(name) => {
                 let mut base_url = config.channel_alias.clone();
                 if let Ok(mut segments) = base_url.path_segments_mut() {
                     segments.push(&name);
                 }
-                base_url
+                add_trailing_slash(&base_url).into_owned()
```

Comment:
This line is a duplicate of the line below. Please return something from the match and then apply removing the trailing slash on the return value.

Reply:
Cleaned it up